doc.go

```diff
@@ -0,0 +1,12 @@
+/*
+Package gotwtr provides a Client for the Twitter v2 APIs.
+
+▲ [CAUTION]
+This library will cover all Twitter APIs v2 in the future,
+but please note that not all Twitter APIs are currently compatible with v2,
+so you cannot use all endpoints that were available in v1.
+
+You can check the current implementation status by clicking on the link below.
+https://github.com/sivchari/gotwtr/issues/8
+*/
```

Comment:
This library will cover all Twitter v2 API in the future, but please note that not all Twitter API are currently compatible with v2.
Because, Twitter announced Twitter v2 API is ready for prime time.
FYI
https://twittercommunity.com/t/ushering-in-a-new-era-for-the-twitter-developer-platform-with-the-twitter-api-v2/162087
So, we covers only Twitter v2 API supported by OAuth 2.0 Bearer Token. 
We will had worked on it, when new one is be handled OAuth 2.0.

Reply:
Please see above it.

Reply:
lgtm, I'll change it to that sentence.